src/js/stores/AppFormStore.js

```diff
@@ -173,7 +173,8 @@ const responseAttributePathToFieldIdMap = {
   "/labels": "labels",
   "/ports": "ports",
   "/uris": "uris",
-  "/user": "user"
+  "/user": "user",
+  "value": "general"
```

Comment:
@pierlo-upitup thanks for pointing that out. The `value` is actually somewhat intentional as described in #3241. [(working link for 3241)](https://github.com/mesosphere/marathon/issues/3241) I feel uneasy by overwriting it on the frontend side like this.